go.mod

```
@@ -31,3 +31,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
```

So before we used, by proxy, commit `48d3d31d256d` of github.com/keybase/go-keychain. This commit is quite old. Looking at their history. Let's use their latest master `f65a47cbe0b120345b64ba52303ebced28b64112`.

```suggestion
replace github.com/keybase/go-keychain => github.com/keybase/go-keychain v0.0.0-20191220220820-f65a47cbe0b1
```